### ISSUE TITLE
Cannot tab onto a date using keyboard

### ISSUE THREAD

**Issue by inghamn** [City-of-Bloomington/timetrack#47]
Right now, you cannot tab into the calendar using the keyboard.  The tab order completely bypasses the entire calendar.

It would be nice to be able to not have to keep reaching for the mouse to do time entry. It would be great if we could tab onto a date link and hit "enter" to trigger the onclick handler for that date. (Launching the time entry modal form)

**Comment by adabutch** [comment]
@inghamn - fyi: time block form should submit via enter now (given values are ok), when not focused on the submit button itself.
